src/qt/transactionview.cpp

```diff
@@ -202,6 +203,9 @@ TransactionView::TransactionView(const PlatformStyle *platformStyle, QWidget *pa
     connect(copyTxPlainText, SIGNAL(triggered()), this, SLOT(copyTxPlainText()));
     connect(editLabelAction, SIGNAL(triggered()), this, SLOT(editLabel()));
     connect(showDetailsAction, SIGNAL(triggered()), this, SLOT(showDetails()));
+
+    // Highlight transaction after fee bump
+    connect(this, SIGNAL(bumpedFee(uint256)), this, SLOT(focusTransaction(uint256)));
```

Comment:
Could use the new connect syntax?

Reply:
@MarcoFalke not sure what you mean by new connect syntax: http://doc.qt.io/qt-5/signalsandslots.html

Reply:
Two options ATM:
```cpp
connect(this, &TransactionView::bumpedFee, [this](const uint256& txid) {
    focusTransaction(txid);
});
```
```cpp
connect(this, &TransactionView::bumpedFee, this, static_cast<void (TransactionView::*)(const uint256&)>(&TransactionView::focusTransaction));
```
Because `TransactionView::focusTransaction` is overloaded.

Reply:
@promag I find the first version more readable and less intimidating. I noticed the other version in your PR for a similar signal.